packages/js/data/src/crud/reducer.ts

```diff
@@ -33,19 +35,37 @@ export const createReducer = () => {
 			data: {},
 			itemsCount: {},
 			errors: {},
+			requesting: {},
 		},
 		payload
 	) => {
+		const itemData = state.data || {};
+
 		if ( payload && 'type' in payload ) {
 			switch ( payload.type ) {
 				case TYPES.CREATE_ITEM_ERROR:
+					const createItemErrorRequestId = getRequestIdentifier(
```

Comment:
Most of the below changes are copy and pasted, with the exception of `requesting` and `getRequestIdentifier`.   The `case TYPES...` had to be separated to allow differences of `requesting` between different actions.